js/focuspoint/jquery.focusPoint.js

```diff
@@ -0,0 +1,435 @@
+(function ($, window, document)
+{
+	"use strict";
+
+	var css = {
+		imageFocus: {
+			self: 'image-focus',
+			wrapper: 'image-focus__wrapper',
+			img: 'image-focus__img',
+			point: 'image-focus__point',
+			clickarea: 'image-focus__clickarea',
+			button: 'image-focus__button'
+		},
+		button: {
+			self: 'button',
+			primary: 'button-primary',
+			disabled: 'button-disabled'
+		}
+	};
+
+	if (!$.imageFocus) {
+		$.imageFocus = {};
+	}
+
+	$.imageFocus.focusPoint = function (el, options)
```

Comment:
It refers to the object on which the plugin is loaded. In this case: `$('.attachment-details')`